src/js/dp/cookies-banner.js

```diff
@@ -10,9 +10,9 @@ const encodedCookiesPolicy = '%7B%22essential%22%3Atrue%2C%22usage%22%3Atrue%7D'
 const cookiesPath = '/'
 
 function determineWhetherToRenderBanner() {
-  const cookiesAreNotSet = !cookiesSet || userIsOnCookiesPreferencesPage()
+  const cookiesAreNotSet = !cookiesSet
 
-  if (cookiesAreNotSet) {
+  if (cookiesAreNotSet && !userIsOnCookiesPreferencesPage()) {
```

Comment:
Can you improve the naming of the conditionals here? (I know you didn't write them...but they are a mix of positive and a false positive and it's taken a whole cuppa coffee to work out what the actual intention is here)

```suggestion
  if (!cookiesSet && !userIsOnCookiesPreferencesPage()) {
```

Reply:
Done.